src/wam2layers/utils/profiling.py

```diff
@@ -109,37 +109,44 @@ def track_stability_correction(
 
         Criteria:
         1. A correction is applied for more than 5% of the grid (first time)
-        2. Previous correction is exceeded by either 5%-point of grid
-        3. Previous correction is doubled in magnitude
+        2. At least 10% more of the grid is corrected compared to the last warning
+        3. Previous strongest correction factor is exceeded by 10%
```

Comment:
Here as well. I would expect this leads to much more warnings. Is that what you wanted?